updateAzaanTimers.py

```diff
@@ -5,68 +5,117 @@
 import sys
 from os.path import dirname, abspath, join as pathjoin
 import argparse
+from configparser import ConfigParser
+
 
 root_dir = dirname(abspath(__file__))
 sys.path.insert(0, pathjoin(root_dir, 'crontab'))
-
-from praytimes import PrayTimes
+from modules.praytimes import PrayTimes
 PT = PrayTimes() 
 
 from crontab import CronTab
 system_cron = CronTab(user='pi')
 
-#HELPER FUNCTIONS
-#---------------------------------
-#---------------------------------
-#Function to add azaan time to cron
+
+# HELPER FUNCTIONS
+# ---------------------------------
+# ---------------------------------
+# Function to add azaan time to cron
 def parseArgs():
     parser = argparse.ArgumentParser(description='Calculate prayer times and install cronjobs to play Adhan')
     parser.add_argument('--lat', type=float, dest='lat',
                         help='Latitude of the location, for example 30.345621')
-    parser.add_argument('--lng', type=float, dest='lng',
+    parser.add_argument('--lon', type=float, dest='lon',
```

Comment:
How about we call it `long`? I believe lat/long is the usually the standard.

Also, could you update [run it for the first time](https://github.com/achaudhry/adhan/blob/master/README.md#run-it-for-the-first-time) section in README to match this? I think it still says `lng`